config/objects/ambitie.yml

```diff
@@ -39,7 +39,7 @@ api:
         - resolver: object_version
           resolver_data:
             path: /version/{object_uuid}
-            response_model: ambitie_full
+            response_model: ambitie_detail
```

Comment:
is dit niet gewoon wat `full` in hield?

Reply:
hier kunnen we wel gewoon die _full voor gebruiken, deze query die erbij komt is zo ongeveer de snelste die er is